java/sha-version.txt

```diff
@@ -0,0 +1 @@
+ArtifactId: anaplan-connect | Version: 3.0.0 | SHA-CommitId: d5196c86da7a335772243af6338c0b9028f8f46b
```

Comment:
is this the SHA commit of the code? I think we need the SHA for artefact that will be created from mvn deploy or mvn install.

Reply:
This is the hash of the current commit id, so if you run `git rev-parse HEAD` this id will be returned. I can change the phase to deploy or install to perform this operation upon installation. 
Can you please tell me what you mean by  SHA for artefact?